examples/fragment-performance/test.js

```diff
@@ -0,0 +1,86 @@
+const puppeteer = require('puppeteer');
+const assert = require('assert');
+const http = require('http');
+
+const Tailor = require('../../index');
+const serveFragment = require('../multiple-fragments-with-custom-amd/fragment');
+const { analyseHooks } = require('./hooks');
+
+describe("Fragment performance in browser", () => {
+    let server;
+
+    before(() => {
+        const tailor = new Tailor({
+            templatesPath: __dirname + '/templates',
+            pipeAttributes: attributes => {
+                const timingGroups = attributes['timing-group']
+                    ? attributes['timing-group'].split(',')
+                    : [];
+                const { id, primary } = attributes;
+                return { timingGroups, id, primary: !!(primary || primary === '') };
+            },
+            pipeInstanceName: 'TailorPipe',
+            maxAssetLinks: 3
+        });
+        server = http.createServer((req, res) => {
+            if (req.url === '/favicon.ico') {
+                res.writeHead(200, { 'Content-Type': 'image/x-icon' });
+                return res.end('');
+            }
+            return tailor.requestHandler(req, res);
+        });
+        server.listen(8080);
```

Comment:
test should start only after server is started, 

before(done => server.listen(0, done));